fili-core/src/test/groovy/com/yahoo/bard/webservice/web/JobsApiRequestSpec.groovy

```diff
@@ -121,63 +117,6 @@ class JobsApiRequestSpec extends Specification {
         errorSubscriber.getOnErrorEvents().get(0).getMessage() == "No job found with job ticket IDontExist"
     }
 
-    def "handleBroadcastChannelNotification returns an Observable that replays notification if it was sent before async timeout"() {
-        setup:
-        TestSubscriber<PreResponse> testSubscriber = new TestSubscriber<>()
-
-        when:
-        Observable<PreResponse> preResponseObservable = defaultJobsApiRequest.handleBroadcastChannelNotification("ticket1")
-
-        broadcastChannel.publish("ticket1")
-        preResponseObservable.subscribe(testSubscriber)
-
-        then:
-        testSubscriber.assertReceivedOnNext([ticket1PreResponse])
-    }
```

Comment:
This test is subsumed by tests in the `JobsServletReactiveChainforResultsEndpointSpec` that verify the results come back synchronously if they are ready within the `asyncAfter` timeout.